LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Crown Copyright (Government Digital Service)
```

Comment:
Of course, we’ve also made changes to it in 2017 as well!

Looked around and found http://softwareengineering.stackexchange.com/a/210491, which suggests — based on guidance from the FSF — that we should make it say “Copyright (c) 2016, 2017 Crown Copyright (Government Digital Service)” (and update it for every year in which we make copyrightable changes).